metaflow/plugins/kfp/constants.py

```diff
@@ -14,4 +14,5 @@
 
 # TODO: Switch out of using this role
 S3_BUCKET = "s3://workspace-zillow-analytics-stage/aip/metaflow"
-S3_AWS_ARN = "arn:aws:iam::170606514770:role/dev-zestimate-role"
\ No newline at end of file
+S3_AWS_ARN = "arn:aws:iam::170606514770:role/dev-zestimate-role"
+S3_AWS_REGION = "us-west-2"
```

Comment:
these variables are still Zillow specific... please don't check these in, but require they be set by environment variables, just like `from_conf('METAFLOW_AWS_S3_REGION')` does.

see: https://github.com/zillow/metaflow/blob/e2d6d16ac91b99f9d8e63233a5d2368789a1b4bf/metaflow/metaflow_config.py#L37-L37

Reply:
Removed these. No longer needed